### PR TITLE
Pp 593 implement amplitude api to track events

### PR DESCRIPTION
**Pull request by ValentinaIancu-Gini**

### CODE REVIEW

File: CaptureSDK/GiniCaptureSDK/Sources/GiniCaptureSDK/Core/Tracking/Utilities/IOSSystem.swift [hunk 1]
```diff
@@ -9,6 +9,7 @@ import UIKit
 internal class IOSSystem {
     private let device = UIDevice.current
     let manufacturer: String = "Apple"
+    let platform: String = "iOS"
```
Comment (mrkulik) [inline CaptureSDK/GiniCaptureSDK/Sources/GiniCaptureSDK/Core/Tracking/Utilities/IOSSystem.swift:12]: [Clarification] should we handle iPad OS here?

**Review by mrkulik — COMMENTED** [review]
@ValentinaIancu-Gini looks great!

File: CaptureSDK/GiniCaptureSDK/Sources/GiniCaptureSDK/Core/Tracking/Utilities/IOSSystem.swift (hunk 1 above)
Comment (ValentinaIancu-Gini) [reply CaptureSDK/GiniCaptureSDK/Sources/GiniCaptureSDK/Core/Tracking/Utilities/IOSSystem.swift]: Done. Thank you!

**Review by mrkulik — APPROVED** [review]

**Review by mrkulik — APPROVED** [review]

**Comment by ValentinaIancu-Gini** [comment]
I will merge this since Vlad already tested the ticket for this PR.

**Pull request merged**